src/utils/Processing.ts

```diff
@@ -161,6 +162,75 @@ export function readConfig(input: string): Configuration {
   return config;
 }
 
+export async function updateExternalDependencies(config: Configuration): Promise<boolean> {
+  // only try to update if we got the config from sushi-config.yaml, and not from an IG
+  const changedVersions: Map<string, string> = new Map();
+  if (config.filePath != null && config.dependencies?.length > 0) {
+    const promises = config.dependencies.map(async dep => {
+      // current and dev have special meanings, so don't try to update those dependencies
+      if (dep.version != 'current' && dep.version != 'dev') {
+        try {
+          const res = await axiosGet(`https://packages.fhir.org/${dep.packageId}`);
+          const latestVersion: string = res?.data?.['dist-tags']?.latest;
+          if (latestVersion) {
+            if (dep.version !== latestVersion) {
+              dep.version = latestVersion;
+              changedVersions.set(dep.packageId, dep.version);
+            }
+          } else {
+            logger.info(`Could not determine latest version for package ${dep.packageId}`);
+          }
+        } catch (err) {
+          logger.info(`Could not get version info for package ${dep.packageId}`);
+        }
+      }
+    });
+    await Promise.all(promises);
+    if (changedVersions.size > 0) {
+      // before changing the file, check with the user
+      const continuationChoice = readlineSync.keyIn(
+        [
+          'Updates to dependency versions detected:',
+          ...Array.from(changedVersions.entries()).map(
+            ([packageId, version]) => `- ${packageId}: ${version}`
+          ),
+          'SUSHI can apply these updates to your sushi-config.yaml file.',
+          'This may affect the formatting of your file.',
+          '- [A]pply updates to sushi-config.yaml',
+          '- [Q]uit without applying updates',
+          'Choose one [A,Q]: '
```

Comment:
This is just genuinely a question. My head understands that `[A]pply` and `[Q]uit` are more obviously clear about what actions you are choosing when you type in an answer, but when I was using this feature, my muscle memory wasn't listening to my head, and I kept expecting to type `y` or `n` to choose to update the packages. I think it might be because there are only two options to really choose from that I kept getting confused with my instinct for yes or no options. Do you think it makes sense to just have the typical y/n choice here since we really only have two options? Or is the additional clarity worthwhile and my muscle memory will eventually catch up?

Reply:
I agree with Julia here - Y/N is something I'm also more used to using when accepting changes in the CLI. In regards to the changes here - maybe we can keep the A/Q options while also allowing the user to have the additional option of using Y in lieu of A and N in lieu of Q?

Reply:
Another note - A and Q are very close on the keyboard and can lead to a misinput.